zgw/zaken-api/src/test/resources/config/application.yml

```diff
@@ -20,3 +20,6 @@ valtimo:
             url: #filled in test
             clientId: valtimo_client
             secret: e09b8bc5-5831-4618-ab28-41411304309d
+            detailsTypeUrl: "http://host.docker.internal:8011/api/v1/objecttypes/3e852115-277a-4570-873a-9a64be3aeb34"
```

Comment:
I feel like this property should be named objectTypeUrl.

Reply:
Removed, not used